templates/pages/setup/authentication/setup.html.twig

```diff
@@ -0,0 +1,103 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% set session_get_plural = call('Session::getPluralNumber', []) %}
+
+<form name='form' action='{{ action }}' method='post' data-track-changes='true'>
+    <div class="container-xl">
+        <div class="card">
+            <div class="card-header">
+                <h3 class="card-title"><i class="ti ti-adjustments me-1"></i>{{ __('Authentication setup') }}</h3>
+            </div>
+            <div class="card-body">
+                <div class="row">
+                    {{ fields.dropdownYesNo(
+                        'is_users_auto_add',
+                        CFG_GLPI.is_users_auto_add,
+                        __('Automatically add users from an external authentication source'),
+                    ) }}
+                    {{ fields.dropdownYesNo(
+                        'use_noright_users_add',
+                        CFG_GLPI.use_noright_users_add,
+                        __('Add a user without accreditation from a LDAP directory'),
+                    ) }}
+
+                    {{ fields.dropdownArrayField(
+                        'user_restored_ldap',
+                        CFG_GLPI.user_restored_ldap,
+                        call('AuthLDAP::getLdapRestoredUserActionOptions', []),
```

Comment:
You should pass values into a `user_restored_ldap_choices` variable to prevent usage of a direct call to a PHP method (`call()` was made to be able to ease the move to Twig templates, but its usage should be removed ASAP).